Updated readme.md

olimorris opened the pull request:
Updated the readme to be more readable and to correct spelling.

Also noticed the build tests say this package is broken!

Commit:
Updated readme.md

Updated to add Laravel Elixir and correct spelling and grammar

Commit:
Merge pull request #1 from olimorris/updated-readme

Updated readme.md

Commit:
Updated readme.md

Mulkave commented:
The error is really weird...

```
PHP Fatal error:  Class 'Vinelab\Cdn\Tests\TestCase' not found in /home/travis/build/Vinelab/cdn/tests/Vinelab/Cdn/AssetTest.php on line 15
```

Commit:
Update travis.yml file

Unsure why the tests aren't working on Travis. Trying this to see if it works.

Commit:
Merge pull request #2 from olimorris/olimorris-travis-fix

Update travis.yml file

Commit:
Updated readme.md

Minor fix to make 'Laravel Support' header the same size as the header below the title.

Commit:
Merge pull request #3 from olimorris/olimorris-readme-update

Updated readme.md

Commit:
Update .travis.yml

Changed 'before_script' commands to see if that fixes Travis tests.

Commit:
Merge pull request #4 from olimorris/olimorris-travis-patch-2

Update .travis.yml

Commit:
Update AssetTest.php

I think the issue is due to case sensitive file system on Travis. Tests pass on OSX (case insensitive filesystem) but not on Travis.

Seeing if this works...

Suggest we scrap this pull request as too many commits.

Commit:
Merge pull request #5 from olimorris/olimorris-patch-1

Update AssetTest.php

Mulkave commented:
@olimorris any updates on this from your side?
